data/core/src/commonMain/kotlin/com/tunjid/heron/data/network/models/PostConversions.kt

```diff
@@ -110,6 +100,7 @@ internal fun PostView.post(
                 createdAt = atProtoLabel.cts,
             )
         },
+        embeddedRecord = postEntity.record?.asExternalModel(),
```

Comment:
A `Post.Record` is not the same type as an embedded record. A `Post.Record` is just text. We'd be losing information here.

The `embeddedRecord` type is a `Record` without the `Post` suffix.

What this method should be doing is checking the `PostView` for the record unions, and checking if a quoted post, feed, starter pack or list is present. The embedded record is one of these.